### PR TITLE
Adds the recipe for Idg.

### PR DESCRIPTION
**Pull request by dipietrantonio**

### CODE REVIEW

File: var/spack/repos/builtin/packages/idg/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    version("1.0.0", commit="3322756fb8b6e3bb1fe5293f3e07e40623ff8486")
+    version("0.8.1", commit="a09f3c85094c592f9304fff4c31e920c7592c3c3")
+
+    depends_on("boost")
+    depends_on("fftw-api@3")
+    depends_on("blas")
+
+    def url_for_version(self, version):
+        return ("https://git.astron.nl/RD/idg/-/archive/{0}/idg-{0}.tar.gz".format(version))
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/idg/package.py]: Why setting `url_for_version` instead of setting `url` at the class level?

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/idg/package.py (hunk 1 above)
Comment (pelahi) [reply var/spack/repos/builtin/packages/idg/package.py]: I think this is a hold over from when I originally created the recipe and the url for a given version wasn't this simple.

File: var/spack/repos/builtin/packages/idg/package.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/idg/package.py]: Can you update the license?

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/idg/package.py (hunk 2 above)
Comment (dipietrantonio) [reply var/spack/repos/builtin/packages/idg/package.py]: Done.

**Review by alalazo — APPROVED** [review]

**Pull request merged**